pages/common/conda.md

```diff
@@ -0,0 +1,31 @@
+# conda
+
+> Package, dependency and environment management for any language.
```

Comment:
Absolutely, I just copied and pasted this from the website and chopped off the list of examples. "Programming language" is better.